utils.py

```
@@ -1,25 +1,26 @@
 import os
 import yaml
 
-def get_apikey():
+def get_apikey(service):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)